frontend/src/app/main/data/plugins.cljs

```diff
@@ -144,8 +145,13 @@
     (watch [_ state _]
       (let [user-can-edit? (dm/get-in state [:permissions :can-edit])]
         (when-let [pid (::open-plugin state)]
-          (open-plugin! (preg/get-plugin pid) user-can-edit?)
-          (rx/of #(dissoc % ::open-plugin)))))))
+          (let [plugin (preg/get-plugin pid)]
+            (open-plugin! plugin user-can-edit?)
+            (rx/of (ev/event {::ev/name "start-plugin"
+                              ::ev/origin "workspace:try-out"
```

Comment:
I think the origin here should be simply `workspace`, the `try-out` suffix does not indicates anything relevant here I think.